sympy/geometry/polygon.py

```diff
@@ -146,6 +146,9 @@ def __new__(cls, *args, **kwargs):
 
         vertices = list(nodup)
 
+
+
+
```

Comment:
What is the role of these blank lines?

Reply:
Sorry, they are by mistake. I will remove them.